utils/data-processing-family-task-submitter-container/pom.xml

```diff
@@ -0,0 +1,162 @@
+
+    <dependencies>
+        <dependency>
+            <groupId>com.github.cafdataprocessing</groupId>
+            <artifactId>data-processing-family-task-submitter</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.github.krallin</groupId>
+            <artifactId>tini</artifactId>
+            <scope>runtime</scope>
+            <type>exe</type>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>${fabric8.docker.maven.version}</version>
+                <executions>
+                    <execution>
+                        <id>docker-build</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                        <configuration>
+                            <skip>false</skip>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>docker-push</id>
+                        <phase>deploy</phase>
+                        <goals>
+                            <goal>push</goal>
+                        </goals>
+                        <configuration>
+                            <filter>data-processing-family-task-submitter</filter>
+                        </configuration>
+                    </execution>
+                </executions>
+                <configuration>
+                <autoPull>always</autoPull>
+                    <images>
+                        <!-- Task Submitter Container Image -->
+                        <image>
+                            <alias>data-processing-family-task-submitter</alias>
+                            <name>${cafFamilyTaskSubmitterContainerName}</name>
+                            <build>
+                                <from>cafapi/opensuse-jre8:1.2</from>
+                                <labels>
+                                    <Build.Number>${project.version}</Build.Number>
+                                    <Build.Date>${maven.build.timestamp}</Build.Date>
+                                    <Git.Branch>${git.branch}</Git.Branch>
+                                    <Git.Commit>${git.revision}</Git.Commit>
+                                </labels>
+                                <optimise>true</optimise>
+                                <assembly>
+                                    <basedir>/</basedir>
+                                    <mode>tar</mode>
+                                    <inline>
+                                        <dependencySets>
+                                            <dependencySet>
+                                                <useProjectArtifact>true</useProjectArtifact>
```

Comment:
`useProjectArtifact=true`?  That doesn't seem right since the artifact is a pom - does it cause a warning?

Reply:
Yep there was a warning in the build log. Set to false.